Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -316,19 +317,9 @@ private void OnActionRequest(RequestPerformActionEvent ev, EntitySessionEventArg
 
     private void OnValidate(Entity<ActionComponent> ent, ref ActionValidateEvent args)
     {
-        if (ent.Comp.CheckConsciousness && !_actionBlocker.CanConsciouslyPerformAction(args.User))
-        {
+        if (ent.Comp.CheckConsciousness && !_actionBlocker.CanConsciouslyPerformAction(args.User)
+            || ent.Comp.CheckCanInteract && !_actionBlocker.CanInteract(args.User, null))
```

Comment:
```suggestion
        if ((ent.Comp.CheckConsciousness && !_actionBlocker.CanConsciouslyPerformAction(args.User))
            || (ent.Comp.CheckCanInteract && !_actionBlocker.CanInteract(args.User, null)))
```

Small thing but showing them grouped together is easier for people to follow